rae_description/urdf/rae_ros2_control.urdf.xacro

```diff
@@ -15,18 +15,18 @@
       <param name="enLB">43</param>
       <param name="enRA">46</param>
       <param name="enRB">47</param>
-      <param name="encTicsPerRevL">187</param>
-      <param name="encTicsPerRevR">187</param>
+      <param name="encTicsPerRevL">756</param>
```

Comment:
Can you showcase the difference between old an new version?

Reply:
[You can check out how encoders are polled here](https://github.com/luxonis/rae-ros/blob/8e4a8a1d564e1cda91b14e674dc261494ba164c6/rae_hw/src/rae_motors.cpp#L190) - now they take count from more states leading to 4x better resolution [compared to previous version](https://github.com/luxonis/rae-ros/blob/057f2d05c5e803a692c5e8f5356f72dae8c908fa/rae_hw/src/rae_motors.cpp#L182). It does take more computing power but more responsive control should be worth it.